sdk/spring/spring-cloud-azure-stream-binder-servicebus/src/main/java/com/azure/spring/cloud/stream/binder/servicebus/ServiceBusMessageChannelBinder.java

```diff
@@ -137,10 +132,10 @@ protected MessageProducer createConsumerEndpoint(ConsumerDestination destination
                     buildCheckpointConfig(properties));
```

Comment:
Seems like here we can adjust the ServiceBusInboundChannelAdapter constructor to always receives destination and group, and decide whether it's a queue internally?